### PR TITLE
Lu 6839/add ips

### PR DESCRIPTION
**Pull request by mattprice-28**
ip list variable in form: ["ip1", "ip2",  ... "ipn"]

### CODE REVIEW

File: aws_infrastructure/eks_cluster/variables.tf [hunk 1]
```diff
@@ -48,4 +48,6 @@ variable "route53_zone" {
 data "aws_caller_identity" "current" {}
 
 
+variable "ONSVPN" {
```
Comment (RyanAldy) [inline aws_infrastructure/eks_cluster/variables.tf:51]: Don't we have to put:      type = list(string) here? Or does the new Terraform version dynamically work out the type when you put the list of IP's in?

Comment (mattprice-28) [reply aws_infrastructure/eks_cluster/variables.tf:51]: Yeah I did it that way but it had a deprecated warning so I took it back out

Comment (RyanAldy) [reply aws_infrastructure/eks_cluster/variables.tf:51]: Cool, thanks.

File: aws_infrastructure/eks_cluster/eks.tf [hunk 2]
```diff
@@ -205,7 +205,6 @@ resource "aws_launch_configuration" "tf_eks" {
   name_prefix                 = "terraform-eks"
   security_groups             = ["${aws_security_group.tf-eks-node.id}"]
   user_data_base64            = "${base64encode(local.tf-eks-node-userdata)}"
-  key_name                    = "${var.environment_name}-${var.user}-vpc-key"
```
[the remark this answers is not in the document]
Comment (ranjit-das) [reply aws_infrastructure/eks_cluster/eks.tf:208]: In Dev standup discussed and agreed that can be removed.

**Comment by ranjit-das** [comment]
Tested and it has added the ons ips from param file (param ONSVPN) into node sg as expected.

**Pull request merged**